packages/cli/src/commands/sandbox/logs.ts

```diff
@@ -31,7 +31,7 @@ export function waitForSandboxEnd(sandboxID: string) {
         break
       }
 
-      const response = await listSandboxes()
+      const response = await listSandboxes({ state: ['running'] })
```

Comment:
I think we can check this more efficiently now, you can do sandbox get, right?

Reply:
`Sandbox.getInfo` is not in the beta branch